hack/generate-configmap.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+# This script generates configmap/s set by arguments
+#
+# To create a CI configmap use the "--ci" option
+# To create an RHEL configmap use the "--rhel" option
+# To create both configmaps use the "--all" option or pass no arguments
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+# Call the function teardown() on the exit of the script
+trap teardown EXIT
+
+teardown() {
+	rm -rf "${TEMP_DIR}"
+}
+
+TEMP_DIR=$(mktemp -d -t keys-XXXXXXXX)
+CONFIGMAP_FILENAME="0000_90_cluster-update-keys_configmap.yaml"
+
+# Generates a public key specified by parameters in the TEMP_DIR directory
+#
+# $1 - Name of the key which will be generated
+# $2 - Verifier public keys separated by internal field separator
+#
+generate_public_key() {
+	truncate -s 0 "${TEMP_DIR}/key.gpg" # Empty the ${TEMP_DIR}key.gpg file
+	for key in ${2}; do
```

Comment:
nit: this unquoted expansion leaves you vulnerable to accidental space injection and sensitive characters and other things.  Probably better to use:

```sh
OUTPUT_NAME="${1}"
shift
for KEY in "${@}"; do
  ...
```

or some such.

Reply:
I started utilizing the bash arrays more, thanks for the tip. Should have done it in the first place instead of using expansions.

Fixed in https://github.com/openshift/cluster-update-keys/commit/d8197c0d0a485af7bb0b9f423e461f9606989991.